sim_pipeline/gg_lens_pop.py

```diff
@@ -1,5 +1,16 @@
 from sim_pipeline.Pipelines.skypy_pipeline import SkyPyPipeline
-from sim_pipeline.gg_lens import GGLens
+from sim_pipeline.gg_lens import GGLens, theta_e_when_source_infinity
+import numpy as np
+
+def draw_test_area(deflector):
+    """
+    draw a test area around the deflector
+    :param deflector: deflector dictionary
+    :return: test area in arcsec^2
+    """
+    theta_e_infinity = theta_e_when_source_infinity(deflector)
+    test_area = np.pi * (theta_e_infinity*2) ** 2
```

Comment:
perhaps you can stretch the test area by a tiny bit (say a factor of 1.5 such to make sure that for slightly different mass profiles, also the very edge of two image pairs can form?

Reply:
ah yes, never mind. I think 1.5 or even 1.2 times the Einstein radius could be enough already. This could lead to some more speedup